### PR TITLE
[ZH] Add `Score`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

麻烦reviewer在提出修改建议的时候给个理由哦
please leave a reason for me to check when suggesting changes! thanks~
然后鸡蛋里挑骨头，为了改而改的就别提了，我会直接resolve的
please, don't try to pick bones from an egg, or suggest changes just because you can. I'll just ignore and resolve them
我是学文的，你觉得我遣词造句有问题可以拿新华字典解释或者拿相关guideline来怼我，别无中生有哈

### CODE REVIEW

File: wiki/Gameplay/Score/zh.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# 分数
+
+*对于其他用法，请参阅[分数（消歧义)](/wiki/Disambiguation/Score)。*
+
+玩家成功完成一张[谱面](https://github.com/ppy/osu-wiki/blob/master/wiki/Beatmap)所获得的分数，是由玩家在每个[打击物件](https://github.com/ppy/osu-wiki/blob/master/wiki/Gameplay/Hit_object) (Hit Object) 上所取得的[判定](https://github.com/ppy/osu-wiki/blob/master/wiki/Gameplay/Judgement)来决定的。目前正在使用的分数系统可以分成两个大版本。
```
Comment (kj415j45) [inline wiki/Gameplay/Score/zh.md:5]: 链接应使用相对路径

File: wiki/Gameplay/Score/zh.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- [osu!](ScoreV1/osu!)
+- [osu!taiko](ScoreV1/osu!taiko)
+- [osu!catch](ScoreV1/osu!catch)
+- [osu!mania](ScoreV1/osu!mania)
+
+## ScoreV2
+
+ScoreV2 是全新一代的分数系统。它的宗旨是对所有游戏模式的分数系统进行标准化，例如在 1.00x 的分数倍率下，完美分数即为 1,000,000，再加上 osu! 的转盘加分，osu!taiko 的长条加分，osu!catch 的香蕉加分。区别于以往对每个打击物件赋值的计分方式，这种计分方式更专注于每个物件相对于 100 万上限的占比。
+
+除了更好的标准化外，ScoreV2 也是对长谱面连击数较多而导致产生[整数溢出](https://zh.wikipedia.org/wiki/整数溢出)问题的解决方案。因为游戏的总分会被存储为一个 32 位的整数，而 ScoreV1 理论上能给出的分数是无限的，这就会导致分数值超过 32 位整数的理论极限 2,147,483,647，从而让分数计算器倒转产生负数分值（视觉上会体现为分数开始慢慢减少）。在实践中，理论最高分超过 32 位整数限制的长地图取得的成绩会自动使用 ScoreV2 计分。
```
Comment (kj415j45) [inline wiki/Gameplay/Score/zh.md:26]:
```suggestion
除了更好的标准化外，ScoreV2 也是对长谱面连击数较多而导致产生[整数溢出](https://zh.wikipedia.org/wiki/整数溢出)问题的解决方案。因为游戏的总分会被存储为一个 32 位的整数，而 ScoreV1 理论上能给出的分数是无限的，这就会导致分数值超过 32 位整数的理论极限 2,147,483,647，从而让分数计算器倒转产生负数分值（看上去就像是分数开始慢慢减少）。实际上，理论最高分超过该限制的长谱面取得的成绩会自动使用 ScoreV2 计分。
```

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Score/zh.md (hunk 1 above)
Comment (FlyingFish23) [reply wiki/Gameplay/Score/zh.md:5]: 是指把这些GitHub链接换成/wiki/beatmap这种链接形式嘛
其实我也不太清楚怎么就搞成了这种GitHub链接，你不指出来我还没看到🤣
```suggestion
玩家成功完成一张[谱面](/wiki/Beatmap)所获得的分数，是由玩家在每个[打击物件](/wiki/Gameplay/Hit_object) (Hit Object) 上所取得的[判定](/wiki/Gameplay/Judgement)来决定的。目前正在使用的分数系统可以分成两个大版本。
```

File: wiki/Gameplay/Score/zh.md (hunk 2 above)
Comment (FlyingFish23) [reply wiki/Gameplay/Score/zh.md:26]: `看上去...`和`超过该限制`我感觉改的没问题，但是后面这里的话...英文原文是`In practice`，我感觉还是译成`在实践中`，词典对于Practice的第一释义也是`实践`
截屏2022-11-28 13 37 58
`实际上`主要相对`Actually`这个单词吧👀
```suggestion
除了更好的标准化外，ScoreV2 也是对长谱面连击数较多而导致产生[整数溢出](https://zh.wikipedia.org/wiki/整数溢出)问题的解决方案。因为游戏的总分会被存储为一个 32 位的整数，而 ScoreV1 理论上能给出的分数是无限的，这就会导致分数值超过 32 位整数的理论极限 2,147,483,647，从而让分数计算器倒转产生负数分值（看上去就像是分数开始慢慢减少）。在实践中，理论最高分超过该限制的长地图取得的成绩会自动使用 ScoreV2 计分。
```

Comment (FlyingFish23) [reply wiki/Gameplay/Score/zh.md:26]: 对不起，我承认我眼睛有问题，我刚看到In practice的释义是实际上🥲

File: wiki/Gameplay/Score/zh.md (hunk 1 above)
Comment (kj415j45) [reply wiki/Gameplay/Score/zh.md:5]: 对，是这样改

Comment (FlyingFish23) [reply wiki/Gameplay/Score/zh.md:5]: 👌🏿改好了

**Pull request merged**